src/site/stages/build/plugins/download-drupal-assets.js

```diff
@@ -16,7 +16,32 @@ async function downloadFile(
   everythingDownloaded,
 ) {
   const asset = assetsToDownload.shift();
-  const response = await client.proxyFetch(asset.src);
+  const fileOutputPath = path.join(
+    options.cacheDirectory,
+    'drupal/downloads',
+    asset.dest,
+  );
+
+  let response;
+  let retries = 3;
+  while (retries--) {
+    try {
+      // eslint-disable-next-line no-await-in-loop
+      response = await client.proxyFetch(asset.src);
+      break;
+    } catch (e) {
+      if (retries > 0) {
```

Comment:
Minor point, but do we want one attempt and then three retries, or one attempt and then two retries?

Reply:
I was thinking 3 tries total.

Reply:
3 tries makes sense to me too..


But what if after that, we prompted the user via the terminal - `You are out of retries. Do you want to try to win more retries?` And the developer engages in a game of Tic Tac Toe against the front-end build to win more retries

Reply:
Seems good, if it doesn't work after 3, there's probably a larger issue. Or if the actual issue is having to wait longer before retrying, we can adjust the wait time needed

Reply:
Haha now that would be a memorable developer experience. Reminds me of Chrome's dinosaur game

Reply:
I played Cool Spot last night on Sega Genesis and got some inspiration from that lol

Reply:
There _is_ a larger issue. Downloading the drupal assets one at a time directly from Drupal is too slow and the connection is unstable on our local machines. I'm not sure what the best solution is, but adding these retries is probably just a band-aid.